cognite/client/_api/data_modeling/graphql.py

```diff
@@ -45,35 +47,52 @@ def apply_dml(
                 >>> from cognite.client import CogniteClient
                 >>> c = CogniteClient()
                 >>> res = c.data_modeling.graphql.apply_dml(
-                ...     ("mySpace", "myDataModel", "v1"),
+                ...     ("mySpace", "myDataModel", "1"),
                 ...     dml="type MyType { id: String! }"
                 ... )
         """
         data_model_id = DataModelId.load(id)
-        graphql_body = f"""
-            mutation UpsertGraphQlDmlVersion{{
-                upsertGraphQlDmlVersion(
-                    graphQlDmlVersion: {{
-                        space: "{data_model_id.space}",
-                        externalId: "{data_model_id.external_id}",
-                        version: "{data_model_id.version}",
-                        graphQlDml: "{dml}",
-                        previousVersion: "{previous_version}",
-                        name: "{name}",
-                        description: "{description}"
-                    }}
-                ) {{
-                    result {{
+        graphql_body = """
+            mutation UpsertGraphQlDmlVersion($dmCreate: GraphQlDmlVersionUpsert!) {
+                upsertGraphQlDmlVersion(graphQlDmlVersion: $dmCreate) {
+                    errors {
+                        kind
+                        message
+                        hint
+                        location {
+                            start {
+                                line
+                                column
+                            }
+                        }
+                    }
+                    result {
                         space
                         externalId
                         version
                         name
                         description
+                        graphQlDml
                         createdTime
                         lastUpdatedTime
-                    }}
-                }}
-            }}
+                    }
+                }
+            }
         """
-        res = self._post_graphql(url_path="/dml/graphql", query=graphql_body)
+        payload = {
+            "query": graphql_body,
```

Comment:
Save a few bits perhaps (remember to `import textwrap`):
```suggestion
            "query": textwrap.dedent(graphql_body),
```